helix-front/client/app/app.component.spec.ts

```diff
@@ -1,7 +1,8 @@
 import { TestBed, async } from '@angular/core/testing';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 
-import { Angulartics2, Angulartics2Piwik } from 'angulartics2';
+// import { Angulartics2 } from 'angulartics2';
+// import { Angulartics2Piwik } from 'angulartics2/piwik';
```

Comment:
This is temporary and necessary to work around a bug. We will re-enable analytics once the upgrades are complete.